File: 3768/3768.cs
public class Solution {
    public int MinimumInversionCountInSubarra(TreeNode root) {
        if (root == null) return 0;
        return 1 + Math.Max(MinimumInversionCountInSubarra(root.left), MinimumInversionCountInSubarra(root.right));
    }
}

File: 3768/3768.go
func minimumInversionCountInSubarra(root *TreeNode) int {
	if root == nil {
		return 0
	}
	l := minimumInversionCountInSubarra(root.Left)
	r := minimumInversionCountInSubarra(root.Right)
	if l > r {
		return 1 + l
	}
	return 1 + r
}